cdap-app-fabric/src/main/java/co/cask/cdap/app/store/Store.java

```diff
@@ -48,20 +48,9 @@
 
 /**
  * {@link Store} operates on a {@link Program}. It's responsible
- * for managing the non-runtime lifecycle of a {@link Program}
+ * for managing the lifecycle of a {@link Program}.
```

Comment:
so now it only manages the non-runtime lifecycle right? this may increase ambiguity though, should we just change it to manages app/program metadata?